README.md

```diff
@@ -1,22 +1,22 @@
 # A minimal logging API for Go
 
 logr offers an(other) opinion on how Go programs and libraries can do logging
-without becoming coupled to a particular logging implementation,  This is not
-an implementation of logging - it is an API.  In fact it is two APIs with twop
+without becoming coupled to a particular logging implementation.  This is not
+an implementation of logging - it is an API.  In fact it is two APIs with two
 different sets of users.
 
 The `Logger` type is intended for application and library authors.  It provides
 a relatively small API which can be used everywhere you want to emit logs.  It
 defers the actual act of writing logs (to files, to stdout, or whatever) to the
 `LogSink` interface.
 
-The `LogSink` interface is intended for logging library implementors.  It is a
+The `LogSink` interface is intended for logging library implementers.  It is a
 pure interface which can be implemented by to provide the actual logging
 functionality.
 
 This decoupling allows application and library developers to write code in
 terms of `logr.Logger` (which has very low dependency fan-out) while the
-implementation of logging is managed "up stack" (e.g. in or near `main()`).
+implementation of logging is managed "up stack" (e.g. in or near `main()`.)
```

Comment:
Period (full stop) belongs outside these parentheses.

Reply:
Yep, in this case, you are right.  I was over zealous with this change.

Reply:
I've never understood this convention, so I generally don't follow it, myself.  But OK.

Reply:
Your intuition was correct here @thockin. The stop belongs outside the parenthetical expression.